ExampleMod/Content/Projectiles/Minions/ExampleSentryShot.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Content.Projectiles.Minions
+{
+	// The projectile shot by ExampleSentry.
+	// Note that this class inherits from Sparkling ball and changes a few properties, but otherwise behaves very similarly
+	public class ExampleSentryShot : SparklingBall
+	{
```

Comment:
I don't like this. Examples should be isolated and "portable", with no hard dependencies on other ExampleMod content.